index.html

```diff
@@ -55,30 +55,233 @@ <h1>
   </div>
       <!--Work Section-->
       <section class="work">
-        <div class="work-title">
+        <div class="title">
           <h1>
             My Recent Works
           </h1>
```

Comment:
- [ ] Hey @mutonic you have done well implementing the requirements for this project but there are some adjustments you have to make in order to proceed. First is the use of the `<h1>` tag. Only the main title should be wrapped in a `<h1>` tag. you can use h2, h3,... tags for other titles. This improves the accessibility and SEO qualities of your website.